### PR TITLE
[12.0][MIG] stock_inventory_cost_info: Migration to 12.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
* Standard procedure
* Adapted tests

@Tecnativa TT20190

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]
Code LGTM

**Review by ps-tubtim — APPROVED** [review]

File: stock_inventory_cost_info/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from odoo import api, fields, models
+
+
+class InventoryLine(models.Model):
+    _inherit = "stock.inventory.line"
+
+    currency_id = fields.Many2one(
+        string="Currency",
+        related="inventory_id.company_id.currency_id",
+        readonly=True,
```
Comment (sergio-teruel) [inline stock_inventory_cost_info/models/stock_inventory.py:13]: For related fields the default value is True, so you can delete this attribute

**Review by sergio-teruel — APPROVED** [review]
just a little comment

File: stock_inventory_cost_info/models/stock_inventory.py (hunk 1 above)
Comment (pedrobaeza) [reply stock_inventory_cost_info/models/stock_inventory.py:13]: As it works the same, please let me this time to not change it for not having to push + CIs.

**Pull request merged**